pkg/api/frontendsettings_test.go

```diff
@@ -60,7 +80,8 @@ func TestHTTPServer_GetFrontendSettings_hideVersionAnonyomus(t *testing.T) {
 	}
 
 	cfg := setting.NewCfg()
-	m := setupTestEnvironment(t, cfg)
+	m, hs, reset := setupTestEnvironment(t, cfg)
+	defer reset()
```

Comment:
I think it's more standard/idiomatic to use [T.Cleanup](https://golang.org/pkg/testing/#T.Cleanup).

Reply:
Ah. TIL that exists! I’ll use that